README.md

```diff
@@ -1,6 +1,7 @@
 # FastRoute integration for Mezzio
 
 [![Build Status](https://github.com/mezzio/mezzio-fastroute/workflows/Continuous%20Integration/badge.svg)](https://github.com/mezzio/mezzio-fastroute/actions?query=workflow%3A"Continuous+Integration")
+[![Type Coverage](https://shepherd.dev/github/mezzio/mezzio-fastroute/coverage.svg)](https://shepherd.dev/github/mezzio/mezzio-fastroute)
```

Comment:
Are we doing this now?

Reply:
IMO, mostly useful to have the link thru to shepherd.dev to see type coverage over time.

Reply:
Its fine by me, but I know in the past its been requested that adding stuff like this (which should be done on all repos going forward), should get brought up with the TSC. I'd say merge for now and we can mention it next meeting?

Reply:
You're probably right there. Maybe the March one though ;)